steps/calib_cal.cwl.in

```diff
@@ -48,11 +48,13 @@ requirements:
 inputs:
   observation:
     type: Directory
+    format: measurementset
```

Comment:
Likewise, we should use a URL that defines the LOFAR measurement set format. Could be a GitHub wiki page from this repo